Controller/CRUDController.php

```diff
@@ -63,6 +63,9 @@ public function setContainer(ContainerInterface $container = null)
      */
     public function render($view, array $parameters = array(), Response $response = null)
     {
+        if (!$this->isXmlHttpRequest()) {
+            $parameters['breadcrumbs_builder'] = $this->get('sonata.admin.breadcrumbs_builder');
+        }
```

Comment:
I'm talking about twig, not C/C++. :stuck_out_tongue: 

I :-1: for embed controller. Embed controllers use a lot of resources.